### PR TITLE
Reset fvlabel color definitions when segment == null

### PR DESCRIPTION
**Pull request by mini-bomba**
- [x] I agree to license my contribution under LGPL-3.0 **or** my contribution is from another project with a license compatible with LGPL-3.0

To test this pull request, follow the [instructions in the wiki](https://github.com/ajayyy/SponsorBlock/wiki/Testing-a-Pull-Request).

***
Without this, we would try to use the color for an 'undefined' category on the first render pass of the element. It was then immediately re-rendered with a segment set, but DR missed the update, which caused it to stick to that 'undefined' category and the label became transparent.

In short, this is yet another fix/workaround for a DR bug

### CODE REVIEW

File: src/components/CategoryPillComponent.tsx [hunk 1]
```diff
@@ -117,13 +117,13 @@ class CategoryPillComponent extends React.Component<CategoryPillProps, CategoryP
     private getColor(): string {
         // Handled by setCategoryColorCSSVariables() of content.ts
         const category = this.state.segment?.category;
-        return `var(--sb-category-preview-${category}, var(--sb-category-${category}))`;
+        return category == null ? null : `var(--sb-category-preview-${category}, var(--sb-category-${category}))`;
```
Comment (beingfranklin) [inline src/components/CategoryPillComponent.tsx:120]: I would personally use nullish coalescing operator (??) in the code:
       ` return var(--sb-category-text-preview-${category ?? ''}, var(--sb-category-text-${category ?? ''})); `       
Suggesting this approach as its cleaner and more concise. However, if you have a specific reason for using the current implementation, please feel free to share it. :)

Comment (mini-bomba) [reply src/components/CategoryPillComponent.tsx:120]: Sorry, but I don't think this helps fix the bug. The issue wasn't some kind of nullpointer error (ts equivalent of that course). The code already generated rules such as `--sb-category-preview-undefined`. Your code would essentially do the same thing, just generating `--sb-category-preview-` instead. The issue was that such CSS variables aren't defined by the CSS variable generator in content.ts and that this property was quickly changed to a valid/defined one on first initialization of the category pill, but dark reader wasn't able to catch that later update.

Making sure the whole property is null when category is null (therefore removing it from inline css) ensures there is only a single color/backgroundColor property update, which leaves less room for dark reader to screw up.

Comment (beingfranklin) [reply src/components/CategoryPillComponent.tsx:120]: Okay, makes sense 👍
